forge-gui/src/main/java/forge/player/PlayerControllerHuman.java

```diff
@@ -1833,53 +1832,51 @@ public List<AbilitySub> chooseModeForAbility(final SpellAbility sa, List<Ability
     @Override
     public List<String> chooseColors(final String message, final SpellAbility sa, final int min, final int max,
                                      List<String> options) {
-        options = options.stream().map(DeckRecognizer::getLocalisedMagicColorName).collect(Collectors.toList());
-        List<String> choices = getGui().getChoices(message, min, max, options);
-        return choices.stream().map(DeckRecognizer::getColorNameByLocalisedName).collect(Collectors.toList());
+        List<MagicColor.Color> enumOptions = options.stream().map(s -> MagicColor.Color.fromByte(MagicColor.fromName(s))).collect(Collectors.toList());
+        List<MagicColor.Color> enumChoices = getGui().getChoices(message, min, max, enumOptions);
```

Comment:
this almost certainly breaks netplay

Reply:
MagicColor.Color is an Enum with only String keys

i'm pretty sure that works with GUI, but @kevlahnota might test it